src/librustc_resolve/lib.rs

```diff
@@ -864,6 +875,16 @@ enum RibKind<'a> {
 }
 
 /// One local scope.
+///
+/// A rib represents a scope names can live in. Note that these appear in many places, not just
+/// around braces. At any place where the list of accessible names (of the given namespace)
```

Comment:
It's probably better to say it more explicitly - the list of accessible names changes or new restrictions on name accessibility are added.